test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -217,3 +218,31 @@ func TestMeshGatewaySecure(t *testing.T) {
 		})
 	}
 }
+
+// verifyFederation checks that the WAN federation between servers is successful
+// by first checking members are alive from the perspective of both servers.
+// If secure is true, it will also check that the ACL replication is running on the secondary server.
+func verifyFederation(t *testing.T, primaryClient, secondaryClient *api.Client, secure bool) {
+	retrier := &retry.Timer{Timeout: 1 * time.Minute, Wait: 1 * time.Second}
+
+	retry.RunWith(retrier, t, func(r *retry.R) {
+		members, err := primaryClient.Agent().Members(true)
+		require.NoError(r, err)
+		require.Len(r, members, 2)
+		require.Equal(r, members[0].Status, serf.StatusAlive)
```

Comment:
I think go back to the const and fix namespaces_test and then we can ditch the serf dep.

Reply:
Done!